app/views/templates/project.html.slim

```diff
@@ -0,0 +1,16 @@
+.container-fluid
+  .page-header
+    h1 {{ project | projectName }}
+  .row.overview
+    .col-lg-3.col-md-4.col-xs-12
+      .summary.panel.panel-default ng-show='project.id'
+        .panel-heading.clearfix
+          .label.label-info.pull-right tooltip='Project ID' {{ project.id }}
+          h3.panel-title Details
+        .panel-body
+          ul
+            li ng-show='project.description' {{ project.description }}
+            li {{ project.testsCount | number }} tests in the project
+            li {{ project.reportsCount | number }} test runs
+            li Created on {{ project.createdAt | amDateFormat:'llll' }}
```

Comment:
Prefer the more human-friendly and explanatory terms "Creation Date" (used in project list) and "Last Update" for the definition terms.